package.json

```diff
@@ -37,7 +37,8 @@
   "private": true,
   "repository": "immutable/ts-immutable-sdk.git",
   "resolutions": {
-    "@openzeppelin/contracts": "3.4.2-solc-0.7"
+    "@openzeppelin/contracts": "3.4.2-solc-0.7",
+    "responselike": "^2.0.0"
```

Comment:
Seems like this should have a comment explaining why its needed.

Reply:
The `responseLike` resolution is to fix a broken type in one Yarn's dependencies. Upgrading won't fix it, as the latest version of Yarn would still include the broken package, which is used in `cacheable-request`.